src/datamodel.js

```diff
@@ -245,11 +245,13 @@ class DataModel extends Relation {
             { fieldsArr, groupByString, defaultReducer: reducerStore.defaultReducer() },
             reducers
         );
+        persistAncestorDerivation(this, newDataModel);
```

Comment:
`persistDerivation ` and `persistAncestorDerivation` should be only one function

Reply:
Resolved